libs/components/lists/src/lib/modules/repeater/repeater-item.component.html

```diff
@@ -2,7 +2,8 @@
   class="sky-repeater-item sky-padding-even-default"
   [attr.aria-label]="itemName"
   [attr.aria-selected]="selectable ? !!isSelected : undefined"
-  [attr.role]="showInlineForm ? 'form' : (itemRole$ | async)?.item"
```

Comment:
We don't need to make a distinction for inline-form components since it's okay that they are placed in a grid row element.